specification/domainservices/resource-manager/Microsoft.AAD/stable/2022-12-01/domainservices.json

```diff
@@ -0,0 +1,1175 @@
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "etag": {
+          "type": "string",
+          "description": "Resource etag"
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "DomainService": {
+      "description": "Domain service.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DomainServiceProperties",
+          "description": "Domain service properties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "DomainServiceListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DomainService"
+          },
+          "description": "the list of domain services."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token for the next page of results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response from the List Domain Services operation."
+    },
+    "DomainServiceProperties": {
+      "description": "Properties of the Domain Service.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Data Model Version"
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Azure Active Directory Tenant Id"
+        },
+        "domainName": {
+          "type": "string",
+          "description": "The name of the Azure domain that the user would like to deploy Domain Services to."
+        },
+        "deploymentId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Deployment Id"
+        },
+        "syncOwner": {
+          "readOnly": true,
+          "type": "string",
+          "description": "SyncOwner ReplicaSet Id"
+        },
+        "syncApplicationId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Sync Application Id"
```

Comment:
>"Sync Application Id" [](http://example.com/codeflow?start=25&length=21)

Its recomende dthat docs be proper plain English sentences, this is overly brief, and capitalizes Application unnecessarily

Reply:
improved description of the property